packages/ui/src/ui/store/selectors/groups.ts

```diff
@@ -61,10 +73,10 @@ const getGroupsTreeFiltered = createSelector(
         const root = tree[ROOT_GROUP_NAME];
         const predicates = [
             nameFilter &&
-                ((node) => {
-                    return node === root || -1 !== node.name.indexOf(nameFilter);
+                ((node: GroupsTreeNode) => {
+                    return node === root || -1 !== node.name!.indexOf(nameFilter);
                 }),
-        ].filter(Boolean);
+        ].filter(Boolean) as Array<(node: GroupsTreeNode) => boolean>;
```

Comment:
It is better to use `compact` from `lodash` instead of `.filter(Boolean)`, cause it does good inferring of types and you won't need to add expressions like `as ...`.